python/cudf/cudf/benchmarks/get_datasets.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import getopt
+import sys
+import os
+import shutil
+
+# Update url and dir where datasets needs to be copied
+datasets = {
+    "cuio_dataset": [
+        "https://rapidsai-data.s3.us-east-2.amazonaws.com/cudf/benchmark/"
+        "avro_json_datasets.zip",
+        "cudf/benchmarks/cuio_data/",
+    ],
```

Comment:
Should we use a NamedTuple or some other type of named object here to represent this instead of just a Python list?